UpdatedSyntheticDataset/SyntheticDataset2/ImageCreator/settings.py

```diff
@@ -2,10 +2,10 @@ class Settings(object):
     FONT_TYPE = "UpdatedSyntheticDataset/data/fonts/GothamBold.ttf"
     BACKGROUND_DIRECTORY_PATH = "/Users/zyin/Desktop/Synthetic Dataset/Backgrounds"
 
-    SAVE_PATH = "/Users/vtolpegin/Desktop"
-    ANSWERS_DIRECTORY = "/SyntheticDataset"
+    SAVE_PATH = "/Users/zyin/Desktop/Synthetic Dataset"
+    ANSWERS_DIRECTORY = "/Answers"
```

Comment:
You should use relative paths when saving.

For example, maybe have the images saved to the desktop of the user running the script with `~/Desktop/`.

Also, are you sure the pathing for the answers directory is correct? `/Answers` means there's a directory at the root of the boot drive that's called `Answers`.

Reply:
I agree with the relative pathing; much cleaner than what we have been doing thus far. WRT to second comment, I believe this directory is autogenerated and represents the root folder for the generated set of targets.